packages/static-build/src/index.ts

```diff
@@ -408,6 +408,10 @@ export const build: BuildV2 = async ({
     );
     const spawnOpts = getSpawnOptions(meta, nodeVersion);
 
+    if (!spawnOpts.env) {
+      spawnOpts.env = {};
+    }
```

Comment:
This might cause a bug because env undefined means `process.env` if I remember correctly

Reply:
this logic was just moved from the `create-react-app` block lines 421-423. It looks like a necessary step in order to modify `.env.PATH`